ledger/src/blockstore.rs

```diff
@@ -10697,33 +10697,39 @@ pub mod tests {
         solana_logger::setup();
         let slot = 1;
         let num_txs = 20;
+        // primary slot content
         let entries = [make_large_tx_entry(num_txs)];
+        // "conflicting" slot content
+        let entries2 = [make_large_tx_entry(num_txs)];
```

Comment:
I think we actually want to use the same entries so the data is identical.

My understanding on the intention of this test is to verify that we detect the `ErasureMeta` mismatch (which happens based on the coding index) and flag this block as duplicate.

By this same rationale, it's good that you made the Merkles match.

Reply:
The content does not matter in this case anyway. The mismatch is triggered on ErasureMeta path (I checked).